data/qtrvsim.metainfo.xml.in

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2022 David Heidelberg -->
+<component type="desktop-application">
+  <id>@MAIN_PROJECT_NAME_LOWER@</id>
+  <name>@GENERIC_NAME@</name>
+  <summary>RISC-V CPU simulator for education purposes</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  <launchable type="desktop-id">@MAIN_PROJECT_NAME_LOWER@.desktop</launchable>
+  <content_rating type="oars-1.1"/>
+
+  <description>
+    <p>
+      RISC-V processor architecture simulator for education purposes with pipeline and cache visualization.
+    </p>
+  </description>
+
+  <releases>
+    <release version="0.9.1" date="2022-03-01">
+      <description>
+        <p>Bugfixes and cleanup.</p>
+      </description>
+    </release>
+  </releases>
+
+  <screenshots>
+    <screenshot type="default" width="1526" height="888">
+      <image>https://raw.githubusercontent.com/cvut/qtrvsim/polish/data/screenshots/0.png</image>
+    </screenshot>
+    <screenshot type="default" width="1526" height="888">
+      <image>https://raw.githubusercontent.com/cvut/qtrvsim/polish/data/screenshots/1.png</image>
+    </screenshot>
```

Comment:
I would change this to the master branch. Otherwise, it will be awkward with future editing. It will become valid as soon as this PS is merged.

Reply:
Thanks for the updates to both

> For consistency, the new files (except screenshots) should be here: https://github.com/cvut/qtrvsim/tree/master/extras/packaging

To follow the convention setup by @jdupak, I would vote for the placement of the FlatPack specific files into
extras/packaging/flatpack in the master [extras/packaging](https://github.com/cvut/qtrvsim/tree/master/extras/packaging).